scenarios/perf-eval/cluster-automatic/terraform-inputs/azure.tfvars

```
@@ -0,0 +1,22 @@
+scenario_type  = "perf-eval"
+scenario_name  = "api_cli_test"
```

```suggestion
scenario_name  = "cluster-automatic"
```
Scenario name doesn't match with folder name

---

Thanks. That was a leftover from a test